components/Dashboard/EntriesColumn/EntriesColumn.js

```diff
@@ -0,0 +1,37 @@
+import { Grid } from "../Dashboard.styled";
+import Link from "next/link";
+
+export default function EntriesColumn() {
+  const boxes = [
+    { gridColumnEnd: "span 4", gridRowEnd: "span 3" },
+    { gridColumnEnd: "span 4", gridRowEnd: "span 2" },
+    { gridColumnEnd: "span 1", gridRowEnd: "span 1" },
+    { gridColumnEnd: "span 1", gridRowEnd: "span 1" },
+    { gridColumnEnd: "span 1", gridRowEnd: "span 1" },
+    { gridColumnEnd: "span 1", gridRowEnd: "span 1" },
+    { gridColumnEnd: "span 2", gridRowEnd: "span 2" },
+    { gridColumnEnd: "span 2", gridRowEnd: "span 2" },
+    { gridColumnEnd: "span 2", gridRowEnd: "span 2" },
+    { gridColumnEnd: "span 2", gridRowEnd: "span 2" },
+  ];
+
+  return (
+    <Grid>
+      {boxes.map((box, index) => (
+        <Link
+          href="/newentry"
+          key={index}
+          style={{
```

Comment:
Please don't use inline styling, but styled-components instead.

You can style an existing component by extending it with styles. Like this:

```js
const StyledLink = styled(Link)`
some css comes here
`
```

for `gridColumnEnd` and `gridRowEnd` you can work with props to set the values.